synapse/replication/tcp/protocol.py

```diff
@@ -86,6 +83,14 @@
     "synapse_replication_tcp_protocol_close_reason", "", ["reason_type"]
 )
 
+tcp_inbound_commands = Counter(
```

Comment:
I wouldn't mind if these had a name which reflected that they were counters, now that they're becoming part of the public interface to this module. In other words: can you rename them to `tcp_*_commands_counter` ?

(I'll also remind you that the second field is meant to be a description, but happy if you want to leave that for now)